adcm_aio_client/core/objects/cm.py

```diff
@@ -41,7 +47,16 @@ def get_own_path(self: Self) -> Endpoint:
         return ("adcm",)
 
 
-class License(InteractiveObject): ...
+class License(InteractiveObject):
```

Comment:
it's not interactive object (doesn't have an id)

pls sync it with @Starovoitov changes

Reply:
License changes reverted